code/controllers/subsystem/air.dm

```diff
@@ -193,6 +193,7 @@ var/datum/subsystem/air/SSair
 					if(!T.air.check_turf_total(enemy_tile))
 						T.excited = 1
 						active_turfs |= T
+						break
```

Comment:
Previously, a single atmos tile floating in space would be detected four times. (This actually happened; there was a single tile of plasteel out in the middle of Z=2 somewhere.) Now it'll be detected once, consistent with the case where tiles are adjacent to other atmos tiles with differing gas mixes.